src/InboundServer/handlers.js

```diff
@@ -323,21 +383,27 @@ const putTransfersById = async (ctx) => {
     });
 
     ctx.response.status = 200;
+    histTimerEnd({ success: true });
 };
 
 
 /**
  * Handles a PUT /parties/{Type}/{ID}/error request. This is an error response to a GET /parties/{Type}/{ID} request
  */
 const putPartiesByTypeAndIdError = async(ctx) => {
+    const histTimerEnd = Metrics.getHistogram(
+        'inbound_put_parties_type_id_error',
```

Comment:
Any reason that you are not using the information that you set in the Route Map instead....it seems that this is using the same metric names for the event framework?

Reply:
decided not to go with this option as name in route map requires the path to access